app/gen-server/entity/Limit.ts

```diff
@@ -7,23 +7,23 @@ export class Limit extends BaseEntity {
   @PrimaryGeneratedColumn()
   public id: number;
 
-  @Column()
+  @Column({type: Number})
```

Comment:
Without these, I was getting `Uncaught ColumnTypeUndefinedError: Column type for Limit#limit is not defined and cannot be guessed. Make sure you have turned on an "emitDecoratorMetadata": true option in tsconfig.json. Also make sure you have imported "reflect-metadata" on top of the main entry file in your application (before any entity imported).If you are using JavaScript instead of TypeScript you must explicitly provide a column type.`

I don't know what makes this work in regular grist but not grist-static. I tried importing reflect-metadata in various places to no avail, but it's very likely doable.

Reply:
I'm also unsure about what is going on here, and have just gone with the flow and added types to other entities.